shared/constants/app.js

```diff
@@ -70,3 +70,7 @@ export const POLLING_TOKEN_ENVIRONMENT_TYPES = {
   [ENVIRONMENT_TYPE_NOTIFICATION]: 'notificationGasPollTokens',
   [ENVIRONMENT_TYPE_FULLSCREEN]: 'fullScreenGasPollTokens',
 };
+
+export const METAMASK_PROD_CHROME_ID = 'nkbihfbeogaeaoehlefnkodbefgpgknn';
+export const METAMASK_FLASK_CHROME_ID = 'ljfoeinjpaedjfecbmggjgodbgkmjkjk';
```

Comment:
This is not the Firefox ID. On Firefox the IDs are set in the manifest, ours is here: https://github.com/MetaMask/metamask-extension/blob/develop/app/manifest/firefox.json#L4

Firefox uses a GUID generated on install as the actual hostname of the extension, so this GUID will be different on each install and thus useless for identification.

Reply:
We already have a Flask ID as well, here: https://github.com/MetaMask/metamask-extension/blob/develop/app/build-types/flask/manifest/firefox.json#L4

Reply:
Also beta, which is here: https://github.com/MetaMask/metamask-extension/blob/develop/app/build-types/beta/manifest/firefox.json#L4

The Chrome ID for the beta is `pbbkamfgmaedccnfkmjcofcecjhfgldn`

Reply:
So this cannot work in Firefox, is that right?

Reply:
Thanks, added Beta for Chrome

Reply:
@hivtushok According to the docs it should work on FF, as it is valid to use the ID that the extension has provided for itself:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/sendMessage#parameters

Reply:
@Gudahtt do you know how can we extract the Firefox GUID to send messages? Or can we use the manifest ids for that?

Reply:
We should be able to use the manifest ID according to the docs!